src/app/common/entities/button.ts

```diff
@@ -0,0 +1,4 @@
+export class Button {
```

Comment:
Create `button.model.ts` in `/shared/components/button` and put this class there
This entity refers to shared component